apps/webpack.js

```diff
@@ -321,16 +321,17 @@ function create(options) {
   var optimization = options.optimization;
   var mode = options.mode;
 
+  // When debugging minified code, use the .js suffix (rather than .min.js)
+  // to allow the application to load minified js locally without running it
+  // through the rails asset pipeline. This is much simpler than hacking the
+  // application to load .min.js locally.
```

Comment:
This comment moved; is it still accurate? Specifically wondering if the "without running it through the rails asset pipeline" comment still makes sense.

Reply:
great catch! it is no longer necessary to run the rails asset pipeline in order to use minified, digested js locally. I will update these comments.

Reply:
Thanks again Brad, I was able to remove some functionality related to debugMinify. I also made some major updates to the instructions and documentation in build.md. You can see them on this commit which I have added to the next PR in the sequence: 28b7d5ca397

Reply:
Hi Brad, apologies for the rebase. Bumping this comment thread to see if the updates in 28b7d5c seem sufficient.

Reply:
Yes! Awesome update to our documentation.  Reading through that clarified a lot of things in my mental model of what we're changing.  Thank you!